### PR TITLE
Fix heritability reports

### PR DESCRIPTION
**Pull request by vlagrassa**
- Minor updates to reports page so it still displays even with missing data
- Revert to accepting any trait name as second column header (in input data)

### CODE REVIEW

File: src/modules/site-v2/base/views/tools/heritability_calculator.py [hunk 1]
```diff
@@ -239,7 +239,7 @@ def report(id):
     ready = True
 
   # If report hasn't finished computing yet, display a waiting page
-  except UnfinishedReportError:
+  except UnfinishedReportError as ex:
     data, result = ex.data, None
```
Comment (r-vieira) [inline src/modules/site-v2/base/views/tools/heritability_calculator.py]: @vlagrassa is there a less convoluted way (non-exception-based) way of passing data to the frontend in this code path?

**Review by r-vieira — APPROVED** [review]
Looks good -- see inline comment about populating the data from the exception.

File: src/modules/site-v2/base/views/tools/heritability_calculator.py (hunk 1 above)
Comment (vlagrassa) [reply src/modules/site-v2/base/views/tools/heritability_calculator.py]: Fixed in e67d52b0afcffaa16bb95778160f86fb855e7fde.  Waiting on PR to figure out empty Indel Finder results.

**Review by r-vieira — APPROVED** [review]
looks good!

**Pull request merged**